src/plugins/vuetify.ts

```diff
@@ -21,7 +21,7 @@ export function getVuetify () {
           error: '#d3272c', // same as $app-red
           info: '#2196F3',
           success: '#1a9031', // same as $app-green
-          warning: '#FFC107'
+          warning: '#FF9800'
```

Comment:
Is this used anywhere else? (eg, button colors)

Reply:
No, the only warning is in App.vue